frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -456,6 +457,21 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                                 >
                                                     Delete feature flag
                                                 </LemonButton>
+                                                <LemonDivider vertical />
+                                                {featureFlags[FEATURE_FLAGS.RECORDINGS_ON_FEATURE_FLAGS] && (
+                                                    <LemonButton
+                                                        to={urls.sessionRecordings(SessionRecordingsTabs.Recent, {
+                                                            events: defaultEntityFilterOnFlag(featureFlag.key).events,
+                                                        })}
+                                                        type="secondary"
+                                                        sideIcon={<IconPlayCircle />}
```

Comment:
I don't think we need the icon here since "View recordings" is explanatory enough~ Plus it makes the button way longer than it needs to be since we also have a "beta" tag on it

Reply:
I also don't love the placement of the view recordings button since the upper right corner is sort of a "action on the flag" area but unsure of where else it would make sense in right now. The best option is to eventually fold the "edit/delete" flag buttons into a dropdown or something similar if we continue to add other actions to the flag